ios/Utils/RNSVGPercentageConverter.m

```diff
@@ -47,6 +47,9 @@ + (CGFloat)percentageToFloat:(NSString *)percentage relative:(CGFloat)relative o
 
 + (BOOL)isPercentage:(NSString *) string
 {
+    if (![string isKindOfClass:[NSString class]]) {
```

Comment:
The Param is already being cast as a string, this would always be false, yeah?

Reply:
No, `isKindOfClass` checks the actual class of the receiving object at runtime rather than examining the type of the pointer. So this would catch the following case:
```objc
id notAString = @(1234);
BOOL result = [RNSVGPercentageConverter isPercentage:notAString];
```

Reply:
fwiw, we are seeing a nontrivial number of crashes in Expo as a result of people passing the wrong type here, so that's how we caught this and that was the motivation for this PR.